docs/source/tutorials/projects.rst

```diff
@@ -0,0 +1,1213 @@
+
+.. code-block::
+
+    from encord.client import EncordClient
+
+    client = EncordClient.initialise("<project_id>", "<project_api_key>")
+
+
+Getting label rows
+------------------
+
+Project label row IDs (``<label_hash>`` uid) are found in the project information Python JSON (``client.get_project()``), which also contain information about the data title (``data_title``), data type (``data_type``) and label status (``label_status``).
+
+.. code-block::
+
+    {
+        ...,
+        'label_rows': [{
+            'label_hash': 'sample_label_uid',
+            'data_hash': 'sample_data_uid',
+            'dataset_hash': 'sample_dataset_uid',
+            'dataset_title': 'sample_dataset_title',
+            'data_title': 'sample_data.mp4',
+            'data_type': 'VIDEO',
+            'label_status': 'LABELLED',
+        }, {
+            'label_hash': ...,
+        }]
+    }
+
+
+Use the client to fetch individual label objects.
+
+.. code-block::
+
+    label = client.get_label_row('sample_label_uid')
+
+Use the ``project.get_labels_list()`` method to get a list each label ID (``<label_hash>`` uid) in a project and conveniently fetch all project labels.
+
+.. code-block::
+
+    project = client.get_project()
+
+    label_rows = []
+    for label_hash in project.get_labels_list():
+        lb = client.get_label_row(label_hash)
+        label_rows.append(lb)
+
+The label row object contains data units with signed URL's (``data_link``) to the labeled data asset.
+
+
+.. code-block::
+
+    {
+        'label_hash': 'sample_label_uid',
+        'data_hash': 'sample_data_uid',
+        'data_type': 'sample_data_type',
+        'data_units': {
+            ...,
+            'sample_data_hash': {
+                'data_hash': 'sample_data_hash',
+                'data_title': 'sample_data_title',
+                'data_link': 'sample_data_link',
+                'data_type': 'sample_data_type',
+                'data_fps': 'sample_dat_fps' # For video
+                'data_sequence': 'sample_sequence_number',
+                'labels': {
+                    ...
+                },
+            },
+        },
+        'object_answers': {
+            ...
+        },
+        'classification_answers': {
+            ...
+        }
+    }
+
+The objects and classifications answer dictionaries contain classification 'answers' (i.e. attributes that describe the object or classification).
+This is to avoid storing the information at every frame in the label dictionary.
+
+For the ``video`` data type, the ``labels`` dictionary contains a series of frames.
+
+.. code-block::
+
+    {
+        'frame': {
+            'objects': {
+                [{object 1}, {object 2}, ...]
+            }
+            'classifications': {
+                [{classification 1}, {classification 2}, ...]
+            }
+        },
+        ...
+    }
+
+For the ``img_group`` data type, the ``objects`` and ``classifications`` are spread in ``labels``.
```

Comment:
I don't understand this sentence I'm afraid

Reply:
Neither do I. This is legacy content from the old docs.

Reply:
We need to clarify then. Do you know what this means? @denis-cord do you know what this means?

Reply:
This whole section about label (as well as the following) needs a complete brush up. For example, I don't like using uid's for generalizing `project_hash`, `dataset_hash`, `data_hash`, etc. The abstraction just makes everything much more unclear. So no need to investigate a lot of time in this now. I think we can just remove the sentence for now.

Reply:
ok